Reorganisation of the proxy page

filippoweb3 opened the pull request:
This PR solves [this](https://github.com/w3f/polkadot-wiki/issues/3710) issue.

- [x] Make new video tutorial about anonymous proxies

Video tutorials will be added to the page once reviewed and published on the youtube channel.

Due to the renaming of anonymous proxies to **pure proxies**, this PR is also connected to:
- Substrate issue [7735](https://github.com/paritytech/substrate/issues/7735#issuecomment-924315254)
- Substrate PR [12283](https://github.com/paritytech/substrate/pull/12283)
- Polkdadot forum [Post #14](https://forum.polkadot.network/t/parachain-technical-summit-next-steps/51/14)

**Tasks**

- [x] Add video tutorials multisig made of anons
- [ ] Add video tutorial Controller vs staking proxy

Commit:
Changed subtitle structure

- Anon and time-delayed proxies are not proxy types
- Moved up "Why use a Proxy"
- Deleted "Proxy" from Proxy type subtitle as already clear these are types of proxy

Commit:
Removed how-to guide

- Removed part how-to remove anonymous proxies
- Added links to support pages
- Rearranged subtitles

Commit:
Removed how-to guide for removing proxies

How-to guide for removing proxies already present in the support docs, added link to the dedicated article

Commit:
Corrected wrong statements

The primary account is accessible, is the proxy that is not more accessible because there is no private key.

Commit:
Added link substrate pallets, small edits to intro

Commit:
Moved anon proxies at the end

Commit:
Changed position of how-to guides for anon

Commit:
Removed multi-sig proxy diagram

This diagram is misleading, anon proxies cannot be included in a multisig

Commit:
Removed image anon proxy diagram

This image can be more clear.

With this commit also removed:
- multisig diagram image
- anon proxy digram

Commit:
Moved video up in the section, changed text

- moved the video before the text (note the video shows misleading info about permissionless management). it should be replaced with a newer video.

- Changed text

- Changed section title

Commit:
Added text to compare proxies with Anon

Commit:
More detailed text added

Commit:
Deleted resources section

This section is not really important here in my opinion, links are useful within the text

Commit:
Added danger banner

- danger banner for anon proxies
- warning for inclusion of anon proxies within multisig

Commit:
Added benefits of Anon

Commit:
More about anon benefits

filippoweb3 commented:
@DrW3RK the content added in the Anonymous proxies section goes together with the video tutorial I will produce. Thus, the video script is in line with what added in the wiki. The video slides (once reviewed) can be embedded within the this page to support what I wrote. The video tutorial can be attached to the page, as the previous one contains misleading information (i.e. addition of anon proxy within a multi-sig etc.). Let me know what do you think. :)

filippoweb3 commented:
@laboon I added you here since we discussed this topic together in the office. Maybe you have something to add?

laboon commented on docs/learn/learn-proxies.md:
```diff
@@ -8,18 +8,30 @@ slug: ../learn-proxies
-Check out our Polkadot YouTube video that explains what proxy accounts are:
+Much like controller accounts in
+[staking](learn-staking.md), proxies allow users to use an account (it can be a stash account in cold storage or another account in a hot wallet) less frequently but actively
+participate in the network with the weight of the tokens in that account. Proxies can be viewed as an "extreme" version of a controller account, i.e. proxies are allowed to perform a limited amount of actions related to specific [substrate pallets](https://docs.substrate.io/reference/frame-pallets/) on behalf of another account. Check out the video below about proxies.
```
```suggestion
participate in the network with the weight of the tokens in that account. Proxies can be viewed as a more powerful and flexible version of a controller account, i.e. proxies are allowed to perform a limited amount of actions related to specific [substrate pallets](https://docs.substrate.io/reference/frame-pallets/) on behalf of another account. Check out the video below about proxies.
```

laboon replied:
Also please change the last sentence to something like "You can view the video below for more information on proxies and see how they are used" (depending on what exactly is in the video).

In general, we want to avoid extremely casual language like "check out" meaning "see".

filippoweb3 replied:
Thank you Bill, this is very helpful to know as many pages have this kind of informal jargon.

laboon commented on docs/learn/learn-proxies.md:
```diff
@@ -8,18 +8,30 @@ slug: ../learn-proxies
+
+## Why use a Proxy?
+
+Proxies are great to use for specific purposes because they add in a layer of security. Rather than
```
```suggestion
Proxies are useful because they add in a layer of security. Rather than
```

laboon commented on docs/learn/learn-proxies.md:
```diff
@@ -8,18 +8,30 @@ slug: ../learn-proxies
+## Why use a Proxy?
+
+Proxies are great to use for specific purposes because they add in a layer of security. Rather than
+using funds in one sole account, smaller accounts with unique roles complete tasks for the main
```
```suggestion
using funds in a single account, smaller accounts with unique roles complete tasks for the main
```

laboon commented on docs/learn/learn-proxies.md:
```diff
@@ -8,18 +8,30 @@ slug: ../learn-proxies
+
+Proxies are great to use for specific purposes because they add in a layer of security. Rather than
+using funds in one sole account, smaller accounts with unique roles complete tasks for the main
+stash account. This drives attention away from the main account and to proxies.
```
Part of this is attention, but the main idea here is that proxies can be "hotter" than the initial account (stash), which can be kept cold but the "weight" of the tokens can be still be used by the hotter accounts.

filippoweb3 replied:
Will add this in the text

laboon commented on docs/learn/learn-proxies.md:
```diff
@@ -86,122 +98,14 @@ proxy can do. Before participating in a crowdloan using an Auction proxy, it is
 
-These complex proxy setups should only be performed if you are comfortable enough interacting directly with 
-the chain, as you will be unable to sign extrinsics using the UI.
+The remove proxies read the section "Removing Proxies" on [this support page](https://support.polkadot.network/support/solutions/articles/65000182179-how-to-create-a-proxy-account).
```
Ungrammatical, please re-word

filippoweb3 replied:
Done

laboon commented on docs/learn/learn-proxies.md:
```diff
@@ -273,5 +177,71 @@ The required deposit amount for one proxy is equal to:
+
+:::info
+
+To create an **anonymous proxy account** check out [this support article](https://support.polkadot.network/support/solutions/articles/65000182196).
```
```suggestion
To create an **anonymous proxy account** see [this support article](https://support.polkadot.network/support/solutions/articles/65000182196).
```

laboon commented on docs/learn/learn-proxies.md:
```diff
@@ -273,5 +177,71 @@ The required deposit amount for one proxy is equal to:
+
+:::
+
+Anonymous proxies are not real proxies. Proxies that we described so far are _existing accounts_ assigned as proxies by a primary account. These proxies act on behalf of the primary account reducing the exposure of the primary account's private key. Remember, the more often we use an account's private key to sign for different things, the more we expose such key to the internet increasing the vis ...
```
"are not real proxies"

I would say "they are very different from other proxy types".

also change "such key" to "that key" and add a comma after "internet".

laboon replied:
There are actually a few more grammatical errors here - I believe we have Grammarly access, please talk to Radha to get it installed.

filippoweb3 replied:
Asked in the Element chat about Grammarly. The other points are fixed now.

laboon commented on docs/learn/learn-proxies.md:
```diff
@@ -273,5 +177,71 @@ The required deposit amount for one proxy is equal to:
+
+Anonymous proxies are not real proxies. Proxies that we described so far are _existing accounts_ assigned as proxies by a primary account. These proxies act on behalf of the primary account reducing the exposure of the primary account's private key. Remember, the more often we use an account's private key to sign for different things, the more we expose such key to the internet increasing the vis ...
+
+Anonymous proxies are new accounts that are _created_ (not assigned) by a primary account. That primary account then acts as `any proxy` on behalf of the anonymous proxy. Note that, to avoid confusion, in this section we will refer to `Anon` for anonymous proxy. Anon are not real proxies, they are **key-less anonymous accounts** as they do have an address but do not have a corresponding private k ...
```
"A-nonymous" means "without a name", so the "belonging" part doesn't really belong here. I have always seen them as more like "anonymous functions" - https://en.wikipedia.org/wiki/Anonymous_function

filippoweb3 replied:
Removed the belonging part

laboon commented on docs/learn/learn-proxies.md:
```diff
@@ -273,5 +177,71 @@ The required deposit amount for one proxy is equal to:
+
+Anonymous proxies are new accounts that are _created_ (not assigned) by a primary account. That primary account then acts as `any proxy` on behalf of the anonymous proxy. Note that, to avoid confusion, in this section we will refer to `Anon` for anonymous proxy. Anon are not real proxies, they are **key-less anonymous accounts** as they do have an address but do not have a corresponding private k ...
+
+Anon accounts cannot sign anything because they do not have private keys. As a consequence, Anon cannot act as proxies (which is controversial given their original name), and cannot be included in multi-sig. An Anon account acting as a proxy will not be able to sign anything on behalf of the primary account, and an Anon within a multi-sig will not be able neither to initiate nor to approve any ca ...
```
```suggestion
Anon accounts cannot sign anything because they do not have private keys. As a consequence, Anon cannot act as proxies (which is admittedly confusing given their original name), and cannot be included in multi-sig. An Anon account acting as a proxy will not be able to sign anything on behalf of the primary account, and an Anon within a multi-sig will not be able neither to initiate nor to approve any call.
```

laboon commented on docs/learn/learn-proxies.md:
```diff
@@ -273,5 +177,71 @@ The required deposit amount for one proxy is equal to:
+
+- **Enhanced security**. Anon accounts cannot be stolen because they do not have private keys. The only accounts that have full access to the Anon are `Any` proxies. Security can be further increased if the `Any` proxy is a multi-signature account.
+
+- **Simplified and secure account management at corporate-level**. Probably the greatest benefit of using Anon accounts is for the management of complex account relationships at corporate-level. Let's take for example 3 Board members Alice, Bob and Charlie. They create a multi-signature account `ABC 1` holding the treasury of their company, and they use it to create the anonymous account `Anon`.  ...
```
Remove "-" from "corporate-level"

laboon replied:
I'd also separate this into several paragraphs.

Ideally, you would add a diagram for this, but that's not necessary for this PR.

filippoweb3 replied:
Split into different paragraphs.

Yes I was planning to add a diagram based on the slides of the new tutorial video.

laboon commented on docs/learn/learn-proxies.md:
```diff
@@ -273,5 +177,71 @@ The required deposit amount for one proxy is equal to:
+
+## Time-delayed Proxies
+
+We can add an additional layer of security to proxies by giving them a delay time. The delay will be
```
I'd add "approximately" to all of these times, since block time can vary, and blocks for a specific slot can be missed.

filippoweb3 replied:
Done

laboon reviewed (changes requested):
Overall good, just come comments

filippoweb3 committed:
Update docs/learn/learn-proxies.md

Co-authored-by: Bill Laboon <laboon@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-proxies.md

Co-authored-by: Bill Laboon <laboon@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-proxies.md

Co-authored-by: Bill Laboon <laboon@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-proxies.md

Co-authored-by: Bill Laboon <laboon@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-proxies.md

Co-authored-by: Bill Laboon <laboon@users.noreply.github.com>

Commit:
Major edits

- Run grammarly
- Implemented Bill's feedback

Commit:
Small fix

Removed the "belonging part" as suggested by Bill

Commit:
Tiny edit from grammarly

filippoweb3 committed:
Update docs/learn/learn-proxies.md

Co-authored-by: Bill Laboon <laboon@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-proxies.md

Co-authored-by: Bill Laboon <laboon@users.noreply.github.com>

Commit:
Minor edits

- spell check
- terminology check

Commit:
Updated screenshots

- updated screenshot showing how to view your proxies on the developer tab
- added screenshot showing how to view proxies on the Account tab
- added some text to support new figure

Commit:
Removed "Putting it all together"

Title of this section a bit odd, as it contains info that is useful at the beginning of the page. Whole content moved up in the "Why using a proxy" section.

Commit:
Changed "why use a proxy"

- replaced diagram, the old one did not match Polkadot branding colours, and it was difficult to follow with many letters.
- updated text, according to the new figure

Commit:
Moved time-delayed proxies

Moved before anonymous proxies, as anon are probably used less often.

Commit:
Added Figure about anonymous proxies

Comparison between "regular proxies" a and anonymous proxies

Commit:
Minor edits

Simplified title sub-sections

filippoweb3 commented:
@laboon I added some info and figures, rearranged some sections to make the page more easy and pleasant to read. I am following useful feedback I got from another PR on staking. Let me know what do you think. I still have to add a figure and modify the text accordingly for anonymous proxies.

Commit:
Updated font

Updated Inter font in picture proxy vs anon

Commit:
Added figure why anon proxies

This figure is used to explain the benefits of having anonymous proxies vs just a staking proxy

Commit:
Removed technical video about anon proxies

- Remove old video as it presented wrong info, such as anon within multisig
- added new video tutorial

Commit:
Added new tutorial about creating and deleting anon proxies

Commit:
Updated example and figure description

Commit:
Added plot figure

filippoweb3 commented:
@laboon @DrW3RK this PR is good to go in terms of content. Let me know what do you think. After the redenomination of anonymous proxies, the page will need an update.

Commit:
Moved banner removing anon

Commit:
Removed confusing Bold points

Commit:
Changed title structure anon proxies

Commit:
Added part about multisigs and anon

Part about putting an anonymous proxy within a multisig

Commit:
Added new diagram and description

- added new diagram anon in multisigs
- added description for the diagram

Commit:
Added section about proxy calls

- proxy calls
- nested proxy calls

Commit:
Minor edits

Commit:
Added figure and description

- added figure for nested proxy calls
- added figure description about nested proxy calls

Commit:
Added more details to figure description

Commit:
Added some more details

Commit:
Minor edits

Commit:
Renamed figure, added new one

- renamed Figure with 2 proxy calls
- added figure with 3 proxy calls
- added description for new figure

Commit:
Added example multisg with anon proxies

- renamed old figure showing multisig with one proxy
- added figure showing multisig with proxies
- added description for new figure
- minor edits

Commit:
More description details

filippoweb3 commented:
@DrW3RK After some testing on Westend I learned a bit more about proxies. I added some interesting use cases. Let me know what you think!

Commit:
Minor edits

Commit:
Added link to tutorial video

Commit:
Added section to insert tutorial video

Commit:
Corrected figure

Commit:
Moved info banner up

Commit:
Fixed titles capitalization

Commit:
Added link to video tutorial about anons

Commit:
Corrected some misconceptions

profradha commented on docs/learn/learn-proxies.md:
```diff
@@ -8,19 +8,41 @@ slug: ../learn-proxies
-
-Check out our Polkadot YouTube video that explains what proxy accounts are:
+Much like controller accounts in
+[staking](learn-staking.md), proxies allow users to use an account (it can be a stash account in cold storage or another account in a hot wallet) less frequently but actively
```
Linking to the specific section. Also, keeping the introduction paragraph light. 

```suggestion
[staking](learn-staking.md#stash-and-controller-accounts-for-staking), proxies allow users to use an account (it can be in cold storage or a hot wallet) less frequently but actively
```

profradha commented on docs/learn/learn-proxies.md:
```diff
@@ -8,19 +8,41 @@ slug: ../learn-proxies
 
+## Why use a Proxy?
+
+Proxies are useful because they add a layer of security. Rather than
```
```suggestion
Proxies are helpful because they let you delegate efficiently and add a layer of security. Rather than
```

profradha commented on docs/learn/learn-proxies.md:
```diff
@@ -8,19 +8,41 @@ slug: ../learn-proxies
+## Why use a Proxy?
+
+Proxies are useful because they add a layer of security. Rather than
+using funds in a single account, smaller accounts with unique roles complete tasks for the main
```
```suggestion
using funds in a single account, smaller accounts with unique roles can complete tasks on behalf of the main
```

profradha commented on docs/learn/learn-proxies.md:
```diff
@@ -8,19 +8,41 @@ slug: ../learn-proxies
+using funds in a single account, smaller accounts with unique roles complete tasks for the main
+stash account.  Proxies can be _hotter_ than the initial account, which can be kept cold, but the _weight_ of the tokens in the colder account can be used by the hotter accounts. This increases the security of your accounts by minimizing the number of transactions the cold account has to make. This also drives attention away from the stash account, although it is possible to determine the relatio ...
+
+Here below there is an example of how you might use these accounts. Imagine you have one stash account as your primary token-holding account and don't want to access it very often, but you do want to participate in staking to earn staking rewards. You could set one of your existing accounts as a staking proxy for that stash account. Note that here the stash account is also the controller, but it  ...
```
```suggestion
Shown below is an example of how you might use these accounts. Imagine you have one stash account as your primary token-holding account and don't want to access it very often, but you want to participate in staking to earn staking rewards. You could set one of your existing accounts as a staking proxy for that stash account. The stash account is also the controller here, but it does not matter because you will always use your staking proxy to sign staking-related transactions.
```

profradha commented on docs/learn/learn-proxies.md:
```diff
@@ -29,22 +51,22 @@ proxy for the relationship. Polkadot offers:
+account. For example, staking proxies have permission to do only staking-related transactions.
 
-### Any Proxies
+### Any
```
Search Engine Optimization (SEO)

```suggestion
### Any Proxy
```

profradha commented on docs/learn/learn-proxies.md:
```diff
@@ -29,22 +51,22 @@ proxy for the relationship. Polkadot offers:
 frequently than the cold account and is therefore less secure.
 
-### Non-transfer Proxies
+### Non-transfer
```
```suggestion
### Non-transfer Proxy
```

profradha commented on docs/learn/learn-proxies.md:
```diff
@@ -29,22 +51,22 @@ proxy for the relationship. Polkadot offers:
 [balance transfers](learn-balance-transfers.md) (including [vested](learn-DOT.md/#vesting) transfers).
 
-### Governance Proxies
+### Governance
```
```suggestion
### Governance Proxy
```

profradha commented on docs/learn/learn-proxies.md:
```diff
@@ -57,63 +79,101 @@ governance proxies or watch our
 :::
 
-### Staking Proxies
+### Staking
```
```suggestion
### Staking Proxy
```

profradha commented on docs/learn/learn-proxies.md:
```diff
@@ -57,63 +79,101 @@ governance proxies or watch our
+validators to nominate. The stash account still needs to make some transactions such as bonding extra funds or designating a new controller account. A proxy doesn't change the _roles_ of stash and controller accounts but does allow the stash to be accessed even less frequently than using a controller account.
 
-### Identity Judgement Proxies
+### Identity Judgement
```
```suggestion
### Identity Judgement Proxy
```

profradha commented on docs/learn/learn-proxies.md:
```diff
@@ -57,63 +79,101 @@ governance proxies or watch our
 [this page](learn-identity.md#judgements).
 
-### Cancel Proxies
+### Cancel
```
```suggestion
### Cancel Proxy
```

profradha commented on docs/learn/learn-proxies.md:
```diff
@@ -57,63 +79,101 @@ governance proxies or watch our
 announcements.
 
-### Auction Proxies
+### Auction
```
```suggestion
### Auction Proxy
```

profradha commented on docs/learn/learn-proxies.md:
```diff
@@ -57,63 +79,101 @@ governance proxies or watch our
+## Time-delayed Proxies
+
+We can add a layer of security to proxies by giving them a delay time. The delay will be
+quantified in a number of blocks. {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} has approximately 6 seconds of block time. A delay value of 10 will mean 10 blocks, which equals approximately 1 minute of delay.
```
```suggestion
quantified in blocks. {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} has approximately 6 seconds of block time. A delay value of 10 will mean ten blocks, which equals about one minute delay.
```

profradha commented on docs/learn/learn-proxies.md:
```diff
@@ -57,63 +79,101 @@ governance proxies or watch our
+
+We can add a layer of security to proxies by giving them a delay time. The delay will be
+quantified in a number of blocks. {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} has approximately 6 seconds of block time. A delay value of 10 will mean 10 blocks, which equals approximately 1 minute of delay.
+The proxy will announce its intended action and wait for the number
```
```suggestion
The proxy will announce its intended action and will wait for the number
```

profradha commented on docs/learn/learn-proxies.md:
```diff
@@ -127,151 +187,137 @@ the chain, as you will be unable to sign extrinsics using the UI.
+
+:::info Walk-through tutorial video
+
+You can see this video tutorial that goes through the scenarios below. The tutorial requires some familiarity with the Extrinsic Tab of the Polkadot-JS UI.
```
Add the relevant video link here

filippoweb3 replied:
I will produce those short videos very soon

profradha commented on docs/learn/learn-proxies.md:
```diff
@@ -127,151 +187,137 @@ the chain, as you will be unable to sign extrinsics using the UI.
-Alice or Bob or even add in a new account into the anonymous proxy, the anonymous proxy would take
-care of that change. If a multisig wanted to modify itself without an anonymous proxy, a whole new
-multisig would be created.
+As the term suggests, nested proxy calls are proxy calls within proxy calls. Such calls are needed if there are proxied accounts that are proxies themselves. In the example diagram below, Alice has a stash account that has a staking proxy account, Anon. Anon is an anonymous proxy, a proxied account originally spawned by Charly that is now a any proxy of Anon and signs everything on its behalf.
```
```suggestion
As the term suggests, nested proxy calls are proxy calls within proxy calls. Such calls are needed if there are proxied accounts that are proxies themselves. In the example diagram below, Alice has a stash account that has a _staking_ proxy account, _Anon_. _Anon_ is an _anonymous_ proxy, a proxied account originally spawned by Charly that is now an _any_ proxy of _Anon_ and signs everything on its behalf.
```

profradha commented on docs/learn/learn-proxies.md:
```diff
@@ -127,151 +187,137 @@ the chain, as you will be unable to sign extrinsics using the UI.
+![nested proxy calls 2x](../assets/nested_proxy_calls_2x.png)
 
-## How to set up a Proxy
+In this case, for example to bond more funds, Charly needs to submit a `prox.proxy` extrinsic to Anon, which in turn submits a `proxy.proxy` extrinsic to Alice including a `staking.bondExtra` extrinsic call, specifying the number of tokens that need to be bounded. If Charly wants to leave, a new account can take his place as any proxy (before Charly leaves!). There is no need to change the stakin ...
```
```suggestion
For example, to bond more funds, Charly needs to submit a `prox.proxy` extrinsic to Anon, which in turn submits a `proxy.proxy` extrinsic to Alice including a `staking.bondExtra` extrinsic call, specifying the number of tokens that need to be bounded. If Charly wants to leave, a new account can take his place as any proxy (before Charly leaves!). There is no need to change the staking proxy account. Also, Alice is the only one who can remove Anon as a staking proxy, and Anon can only perform staking-related tasks. For example, Anon cannot send funds out from Alice's account.
```

profradha commented on docs/learn/learn-proxies.md:
```diff
@@ -127,151 +187,137 @@ the chain, as you will be unable to sign extrinsics using the UI.
+![nested proxy calls 3x](../assets/nested_proxy_calls_3x.png)
 
-![Remove Proxies](../assets/remove_proxies.png)
+## Anonymous Proxies
```
Add an info not here mentioning that they are now called Pure Proxies and the reasoning/reference links

filippoweb3 replied:
Done

profradha commented on docs/learn/learn-proxies.md:
```diff
@@ -127,151 +187,137 @@ the chain, as you will be unable to sign extrinsics using the UI.
+:::
 
-  ![anon proxy info](../assets/kill-proxy-1.png)
+Anonymous proxies are very different from other proxy types. Proxies that we described so far are _existing accounts_ assigned as proxies by a primary account. These proxies act on behalf of the primary account reducing the exposure of the primary account's private key. Remember, the more often we use an account's private key to sign for different things, the more we expose that key to the intern ...
```
The proxies <-> bodyguards analogy works for the context of security. In general, the analogy of an organization with different departments, Prime minister/President with different ministers for different tasks and privileges is stated.

```suggestion
Anonymous proxies are very different from other proxy types. The proxies we described so far are _existing accounts_ assigned as proxies by a primary account. These proxies act on behalf of the primary account, reducing the exposure of the primary account's private key. Remember, the more often we use an account's private key to sign transactions, the more we expose that key to the internet, increasing the visibility of that account. The purpose of a proxy is thus to draw the attention of potential attackers away from the primary account, as proxies' private keys will be used most of the time to perform actions on behalf of the primary account. We can imagine proxies as bodyguards of a VIP, loyal and ready to risk their lives to ensure the VIP's protection.
```

filippoweb3 replied:
I will more context in the last commits, beyond security

profradha commented on docs/learn/learn-proxies.md:
```diff
@@ -127,151 +187,137 @@ the chain, as you will be unable to sign extrinsics using the UI.
+![anonymous proxies](../assets/proxy_vs_anon.png)
 
-  ![add proxy to delete](../assets/kill-proxy-2.png)
+Anonymous proxies are new accounts that are _created_ (not assigned) by a primary account. That primary account then acts as `any proxy` on behalf of the anonymous proxy. Note that, to avoid confusion, in this section we will refer to `Anon` for anonymous proxy. Anon are **key-less anonymous accounts** as they do have an address but do not have a corresponding private key and, in some sense, are  ...
```
Uniformity when referring to `Anon`

```suggestion
Anonymous proxies are new accounts that are _created_ (not assigned) by a primary account. That primary account then acts as _any_ proxy on behalf of the anonymous proxy. Note that, to avoid confusion, in this section we will refer to _Anon_ for anonymous proxy. _Anon_ are **key-less anonymous accounts** as they do have an address but do not have a corresponding private key and, in some sense, are anonymous as nobody has a private key to control it. The use of the _Anon_ is strictly bound to the relationship between the _Anon_ and the _any_ proxy (not necessarily the one who created the _Anon_ in the first place). Once that relationship is broken, the _Anon_ will be inaccessible (even if visible on the Polkadot-JS UI). Also, _Anon_ accounts are non-deterministic, this means that if we lose one _Anon_, the next one we create from the same primary account will have a different address.
```

profradha commented on docs/learn/learn-proxies.md:
```diff
@@ -127,151 +187,137 @@ the chain, as you will be unable to sign extrinsics using the UI.
 
-- **Step 3**: Go to https://polkadot.js.org/apps/#/extrinsics
-- **Step 4**: Call extrinsic `proxy.killAnonymous` using the selected account ANON PROXY and the following parameters:
+Anon accounts cannot sign anything because they do not have private keys. As a consequence, Anon cannot act as proxies (which is admittedly confusing given their original name), and cannot be included in multi-sig. An Anon account acting as a proxy will not be able to sign anything on behalf of the primary account, and an Anon within a multi-sig will be able neither to initiate nor to approve any ...
```
Removing factually incorrect content

```suggestion
_Anon_ accounts cannot sign anything because they do not have private keys. Although _Anon_ cannot sign any transaction directly, they can act as proxies (or better, proxy channels) within `proxy.proxy` calls. 
```

filippoweb3 replied:
Right! I wrote this before discovering proxy calls.

profradha commented on docs/learn/learn-proxies.md:
```diff
@@ -127,151 +187,137 @@ the chain, as you will be unable to sign extrinsics using the UI.
+:::danger 
 
-  ![call extrinsic](../assets/kill-proxy-3.png)
+Once you remove the relationship with `Any` proxy, the Anon will be inaccessible. Also, Anon accounts cannot sign for anything. For example, do not include Anon in a multi-sig account.
```
```suggestion
Once you remove the relationship with _any_ proxy, the Anon will be inaccessible. Also, Anon accounts cannot sign for anything.
```

profradha commented on docs/learn/learn-proxies.md:
```diff
@@ -127,151 +187,137 @@ the chain, as you will be unable to sign extrinsics using the UI.
+:::caution Removing anonymous proxies
 
-## How to view your Proxies
+The procedure for removing an Anon is different from the one used to remove proxies. Visit the section "Removing an Anonymous Proxy" on [this support article](https://support.polkadot.network/support/solutions/articles/65000182196), or watch [this technical explainer video](https://www.youtube.com/watch?v=T443RcCYP24).
```
```suggestion
The procedure for removing an _Anon_ is different from the one used to remove proxies. Visit the section "Removing an Anonymous Proxy" on [this support article](https://support.polkadot.network/support/solutions/articles/65000182196), or watch [this technical explainer video](https://www.youtube.com/watch?v=T443RcCYP24).
```

profradha commented on docs/learn/learn-proxies.md:
```diff
@@ -127,151 +187,137 @@ the chain, as you will be unable to sign extrinsics using the UI.
+## Why Anonymous Proxies?
 
-## Putting It All Together
+Despite their complexity and associated dangers, Anon accounts have important benefits. We discuss these benefits below.
```
```suggestion
Despite their complexity and associated dangers, _Anon_ accounts have important benefits. We discuss these benefits below.
```

filippoweb3 replied:
I remember Ross or Kian saying to put in `` only code stuff. Maybe we can put it in italic? Just trying to stay consistent. We might add these things in the contribution section so that everybody know? Also the title capitalization you sent me yesterday? This applies to some other comments below.

profradha commented on docs/learn/learn-proxies.md:
```diff
@@ -127,151 +187,137 @@ the chain, as you will be unable to sign extrinsics using the UI.
-and don't want to access it very often, but you do want to participate in governance and staking.
-You could set Governance and Staking proxies.
+### Enhanced Security
+Anon accounts cannot be stolen because they do not have private keys. The only accounts that have full access to the Anon are `Any` proxies. Security can be further increased if the `Any` proxy is a multi-signature account.
```
```suggestion
Anon accounts cannot be stolen because they do not have private keys. The only accounts that have full access to the _Anon_ are _Any_ proxies. Security can be further increased if the _Any_ proxy is a multi-signature account.
```

profradha commented on docs/learn/learn-proxies.md:
```diff
@@ -127,151 +187,137 @@ the chain, as you will be unable to sign extrinsics using the UI.
+:::
 
-## Proxy Deposits
+Probably the greatest benefit of using Anon accounts is the management of complex account relationships at a corporate level. Let's take for example 3 accounts belonging to Charlie, Dan and Eleanor working for Company X. Charlie holds funds belonging to Company X, but he wants to leave the company and transfer the economic responsibility to Eleanor. Dan is a staking proxy of Charlie.
```
```suggestion
Probably the greatest benefit of using _Anon_ accounts is the management of complex account relationships at a corporate level. Let's take for example 3 accounts belonging to Charlie, Dan and Eleanor working for Company X. Charlie holds funds belonging to Company X, but he wants to leave the company and transfer the economic responsibility to Eleanor. Dan is a staking proxy of Charlie.
```

profradha commented on docs/learn/learn-proxies.md:
```diff
@@ -127,151 +187,137 @@ the chain, as you will be unable to sign extrinsics using the UI.
+
+![multisig with one anon](../assets/multisig_with_one_anon.png)
+
+If Charly wants to leave the multisig, a new any proxy can be added to Anon and Charly can be removed (by himself or by the new any proxy). Note that the multisig also contains Bob that in this specific example does not do anything.
```
```suggestion
If Charly wants to leave the multisig, a new _any_ proxy can be added to Anon and Charly can be removed (by himself or by the new any proxy). Note that the multisig also contains Bob that in this specific example does not do anything.
```

profradha commented on docs/learn/learn-proxies.md:
```diff
@@ -127,151 +187,137 @@ the chain, as you will be unable to sign extrinsics using the UI.
+
+:::info Walk-through tutorial video
+
+You can see this video tutorial that goes through this scenario. The tutorial requires some familiarity with the Extrinsic Tab of the Polkadot-JS UI.
```
Add the relevant video link here

DrW3RK reviewed (changes requested):
Good work! The proxy page is comprehensive now. I added a few suggestions improving readability and a few action items for you to complete.

I will make another pass on the PR after the suggestions are incorporated.

filippoweb3 committed:
Update docs/learn/learn-proxies.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-proxies.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-proxies.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-proxies.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-proxies.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-proxies.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-proxies.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-proxies.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-proxies.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-proxies.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-proxies.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-proxies.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-proxies.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-proxies.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-proxies.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-proxies.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-proxies.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-proxies.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-proxies.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-proxies.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-proxies.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-proxies.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-proxies.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-proxies.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

Commit:
SEO stuff

- changed "proxies" to "proxy"

Commit:
Moved VIP stuff at the beginning and added some additional contexts

Commit:
Added italic, details about non-ownership of then anon

Commit:
Added info redenomination anon proxies

Commit:
Tiny edits

Commit:
Prettified

DrW3RK commented:
Approved! You can merge after you add the video links are added

Commit:
Removed old Figures, added new ones (partial)

Commit:
Replaced more figures (partial)

Minor edits to figure descriptions

Commit:
Major edits

- Finished figure replacement
- updated text
- mentioned pure proxy

Commit:
Deleted banner

Commit:
Replaced figure

Commit:
Updated figure description

Commit:
Minor edits to figure description

Commit:
Minor edits

filippoweb3 commented:
hey @laboon let me know what you think about this PR. I still need your approval. I am planning to link just one tutorial showing the difference between staking proxy and controller.

filippoweb3 committed:
Merge branch 'master' into reorg-proxy-page

Commit:
Prettified

Commit:
Reorg proxy deposit

- removed rust code part
- polished text
- prettified

Commit:
Added video pure proxies - multisig
